core/editable.js

```diff
@@ -504,7 +504,7 @@
 				if ( !isSnapshot )
 					data = this.editor.dataProcessor.toHtml( data );
 
-				this.setHtml( data );
+				this.setHtml( CKEDITOR.tools.htmlSafeByReview(data, 'TODO, check if this is safe') );
```

Comment:
Since we don't know if this is safe/easily refactorable let's legacy convert it?

Reply:
Will be legacy converted